workflows/Create_BMC_Incident_for_Top_Remediations_from_InsightVM/workflow.spec.yaml

```
@@ -0,0 +1,16 @@
+extension: workflow
+product: ["insightconnect"]
+name: Create_BMC_Incident_for_Top_Remediations_from_InsightVM
+title: "Create BMC Incident for Top Remediations from InsightVM"
+description: "This workflow will get the top remediations from InsightVM, and create a BMC Remedy ITSM Incident for each remediation. The Incident will have details on the affected endpoints as well as the Fix and risk score."
+version: 1.0.0
+vendor: rapid7
+support: rapid7
+status: []
+hub_tags:
+  use_cases: [application_security, patch_management, remediation_prioritization, remediation_ticketing, soar]
+  keywords: [vulnerability management, bmc]
```

Remove vuln management since it's listed in the use cases, per previous review.

```suggestion
  keywords: [bmc]
```